tests/unit_test/data_processor/data_processor_test.py

```
@@ -15894,7 +15896,7 @@ def test_list_actions(self):
             'email_action': [], 'form_validation_action': [], 'prompt_action': [], 'database_action': [],
             'pyscript_action': [], 'web_search_action': [], 'live_agent_action': [], 'callback_action': [], 'schedule_action': [],
             'utterances': ['utter_offer_help', 'utter_query', 'utter_goodbye', 'utter_feedback', 'utter_default',
-                           'utter_please_rephrase'], 'web_search_action': []}, ignore_order=True)
+                           'utter_please_rephrase'], 'web_search_action': [], 'parallel_action': []}, ignore_order=True)
```

Ruff (0.8.2)

Dictionary key literal `'web_search_action'` repeated

Remove repeated key literal `'web_search_action'`

(F601)